integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSessionMigration.java

```diff
@@ -197,11 +211,43 @@ public void testSessionMigration() {
             "After the primary server stopped, HTTP session state should be migrated to the new primary server")
     );
 
+    finalPrimaryServerName = primaryServerName;
+
     logger.info("SUCCESS --- testSessionMigration \nThe new primary server is {0}, it was {1}. "
         + "\nThe session state was set to {2}, it is migrated to the new primary server.",
             primaryServerName, origPrimaryServerName, SESSION_STATE);
   }
 
+  /**
```

Comment:
Can we merge the test testAnnotationWSlash()  into @Test testSessionMigration() as this is a single line of verification which can be done before verifying the session migration feature.

Reply:
I think keep two separate test cases would be much clearer for the purpose of the testing, plus testAnnotationWSlash needs the diff set of variables

Reply:
Modify the test name to testPodAnnotationWithSlash

Reply:
done